src/gardener/gardener-tree.js

```diff
@@ -37,7 +38,8 @@ const nodeType = {
   NODE_TYPE_LANDSCAPE: 'landscape',
   NODE_TYPE_FOLDER: 'folder',
   NODE_TYPE_CLUSTER_TYPE: 'clusterType',
```

Comment:
rename to something like `NODE_TYPE_PROJECT_RESOURCE` or `NODE_TYPE_PROJECT_SCOPED_RESOURCE`